cmd/shell/cmd/client.go

```diff
@@ -11,6 +19,22 @@ const (
 )
 
 func init() {
+	client.RegisterFlagBinder(func(fs *pflag.FlagSet, namer naming.Namer) {
+		fs.StringVar(&Auth.Username, namer("auth-username"), Auth.Username, "username for authentication")
```

Comment:
Default is "",

Reply:
rename it to `username` and `password`.
Help string should be like this: "username for gRPC basic authentication"

Reply:
Please don't resolve the conversation without explaining why it should get closed.